### ISSUE TITLE
Dynamic tooltips are not handled correctly

### ISSUE THREAD

**Issue by petercmuc** [tinymce/tinymce#9204]
## 📝 Context
In version 6.8.0 tinyMCE shows "dynamic" tooltips for font size, font family, text color and background color containing the currently set value. E.g: "Font sizes 12px" or "Text color Black". In previous versions the tooltips only contained the "Font sizes", or "Text color" resp without dynamic value.

Localizing these tooltips creates a problem:
While you can translate "Font sizes" to something in a different language in your tinyMCE configuration, this will not be reflected in the UI since you need to provide the complete text that tinyMCE shows. So this translation will have no effect.
You would need to provide translations for "Font sizes 12pt", "Font sizes 14pt", and so on.

1) ```
	tinymce.addI18n('en', {
		"Font sizes": "Schriftgröße", // has no effect  
		"Bold": "Fett", // works  
		"Font sizes 14pt": "Schriftgröße 14pt", // works  
	})
```

### ✔️ Expected result
Tooltip "Font sizes 12pt" to be "Schriftgröße 12pt"
Tooltip "Text color Black" to be "Text color black"

### ❌ Actual result
Tooltip "Font sizes 12pt" 
Tooltip "Text color Black" 

### ❓ Possible solution

## 📃 Other details
- First affected version: 6.8.0
- Worked in version: 6.7.x

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by TheSpyder** [comment]
Apologies, we haven't published details of this yet but the new string is fully translatable. If we take font size as an example, here's the diff:
https://github.com/tinymce/tinymce/commit/b00c61c3dcb2079609b381902e03f66b436b0052#diff-11e2472e3101ac41a5e5eb52118c0e5addbab9c4a1ad48bc8327517d71c0fc31R115-R118

and `getTooltipText` points to this code:
https://github.com/tinymce/tinymce/blob/b00c61c3dcb2079609b381902e03f66b436b0052/modules/tinymce/src/themes/silver/main/ts/ui/core/complex/utils/Tooltip.ts#L4

The string changed from `Font sizes` to `Font sizes {0}`. The same applies to all the other dropdowns - the English string has ` {0}` added, and at runtime the `{0}` is replaced with the current value.

**Comment by TheSpyder** [comment]
Looking at this, since we haven't updated the translation source in crowdin yet, we might sneak a change into 6.8.2 to make the string `Font size {0}`. That plural doesn't need to be there anymore, it was missed in code review.

**Comment by petercmuc** [comment]
Hi @TheSpyder 
thanks a lot for the quick reply 👍  That solves my issue.

**Comment by TheSpyder** [comment]
Note that we have decided to treat the plural issue as a bug, and will fix it in 6.8.2 before we update the source strings on crowdin. So keep in mind that your `Font sizes {0}` key will need to change to `Font size {0}` after that (or just translate both for now).

**Comment by TheSpyder** [comment]
TinyMCE 6.8.2 has been released with changes to these strings as I suggested last week, and also a tweak to the `align` dropdown tooltip:
https://github.com/tinymce/tinymce/pull/9213
https://github.com/tinymce/tinymce/pull/9230
